Sources/NavigationBarController.swift

```diff
@@ -137,21 +153,23 @@ class NavigationBarController: UIViewController {
     }
 }
 
-extension NavigationBarController: UICollectionViewDelegate, UICollectionViewDataSource {
+extension NavigationBarController: UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
```

Comment:
Can you split the extension so one extension implements one protocol?

extension NavigationBarController: UICollectionViewDelegate
extension NavigationBarController: UICollectionViewDataSource
extension NavigationBarController: UICollectionViewDelegateFlowLayout